cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -209,12 +209,16 @@ pub enum VecExtendOp {
 pub enum VecALUOp {
     /// Signed saturating add
     SQAddScalar,
+    Sqadd,
```

Comment:
Tiniest possible nit, but: we should be consistent with capitalization, so either `SQAdd` here, or change above to `SqaddScalar`.

Reply:
@akirilov-arm has a patch (not yet on github) that is going to be renaming/moving the Scalar version. Then `Sqadd` will be consistent with the other `VecALUOp` variants.

Reply:
Sounds good, thanks.